### PR TITLE
Standardise .vscode files

### PR DESCRIPTION
**Pull request by marctemp**
https://eaflood.atlassian.net/browse/SFI-1967

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,12 +1,21 @@
 {
   "version": "0.2.0",
-  "configurations": [
-    {
-      "name": "Attach to Node Functions",
+  "configurations": [{
+      "name": "Debug",
```
Comment (marctemp) [inline .vscode/launch.json:4]: Good name for this?

File: .vscode/launch.json [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
       "type": "node",
       "request": "attach",
       "port": 9229,
-      "preLaunchTask": "func: host start"
-    }
-  ]
+      "preLaunchTask": "Function start"
+    },
+    {
+      "name": "Jest debug",
```
Comment (marctemp) [inline .vscode/launch.json:11]: Good name for this?

File: .vscode/tasks.json [hunk 3]
```diff
@@ -1,23 +1,16 @@
 {
   "version": "2.0.0",
-  "tasks": [
-    {
+  "tasks": [{
+      "label": "Function start",
```
Comment (marctemp) [inline .vscode/tasks.json:4]: Good name for this?

File: .vscode/tasks.json [hunk 4]
```diff
@@ -1,23 +1,16 @@
...
       "type": "func",
       "command": "host start",
       "problemMatcher": "$func-node-watch",
       "isBackground": true,
-      "dependsOn": "npm install (functions)"
+      "dependsOn": "Install packages"
     },
     {
+      "label": "Install packages",
```
Comment (marctemp) [inline .vscode/tasks.json:12]: Good name for this?

**Comment by marctemp** [comment]
Judge me and my names

File: .vscode/launch.json (hunk 2 above)
Comment (simonnineteen90) [reply .vscode/launch.json:11]: agree with naming

File: .vscode/tasks.json (hunk 3 above)
Comment (simonnineteen90) [reply .vscode/tasks.json:4]: agree with naming

File: .vscode/tasks.json (hunk 4 above)
Comment (simonnineteen90) [reply .vscode/tasks.json:12]: agree with naming

**Review by simonnineteen90 — APPROVED** [review]
LGTM

**Pull request merged**